### PR TITLE
Feat: Added Image and Input component

### PR DESCRIPTION
**Pull request by JaswanthReddyPadala**
Added:
- Image Component
- Input Component with Validation

image

image

image

### CODE REVIEW

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  cursor: pointer;
+  font-size: 1.2rem;
+}
+
+.radio-input input {
+  margin-right: 0.5rem;
+}
+
+.input-disabled {
+  border-bottom: 2px solid var(--dark-gray);
```
Comment (poornima-alugubelly) [inline components/input/input.css:37]: good naming convention

File: components/input/input.html [hunk 2]
```diff
@@ -56,11 +59,158 @@ <h2 class="text-general docs-sidebar_heading text-24">Components</h2>
 
         <div class="docs-content text-general">
             <h1 class="text-36 mb-1">Inputs</h1>
+            <p class="text-16 mt-2 mb-1">Input are the way to get data from user. There are mutlipe types of input</p>
             <hr>
+            <!-- Primary Inputs-->
+            <div class="docs-content-sub-container">
```
Comment (poornima-alugubelly) [inline components/input/input.html:65]: its good that u have added comments 👍

**Review by poornima-alugubelly — APPROVED** [review]
Good to merge

File: components/image/image.html [hunk 3]
```diff
@@ -1,66 +1,146 @@
...
-                    <li class="docs-sidebar_list-item"><a href="../avatar/avatar.html">Avatar</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../badge/badge.html">Badge</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../button/button.html">Button</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../card/card.html">Card</a></li>
-                    <li class="docs-sidebar_list-item-active"><a href="./image.html">Image</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../input/input.html">Input</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../rating/rating.html">Rating</a></li>
-                    <li class="docs-sidebar_list-item"><a href="../input/input.html">Grid</a></li>
+      <div class="docs-content text-general">
+        <h1 class="text-36">Images</h1>
```
Comment (narharikale) [inline components/image/image.html:94]: can we use `text-size-36` or `font-size-lg` as a classname , it would be more readable

**Review by narharikale — APPROVED** [review]
code is clean ✔

File: components/image/image.html (hunk 3 above)
Comment (JaswanthReddyPadala) [reply components/image/image.html:94]: Okay, I am just following that naming convention. Thanks for the suggestion. I'll refactor this soon

**Pull request merged**